### PR TITLE
chore: add arm64 docker image

### PR DESCRIPTION
**Pull request by sunnyguduru**
Story details: https://app.shortcut.com/launchdarkly/story/239334

Added arm64 image as well. Changed some formatting

### CODE REVIEW

**Comment by cwaldren-ld** [comment]
If you are publishing multiple architectures, it should be as a multi-arch image. That way, they can simply pull ldcli and it will work on their platform. 

This is kinda message in goreleaser, but here's how it's done in Relay: (docker manifest) https://github.com/launchdarkly/ld-relay/blob/v8/.goreleaser.yml#L122

**Comment by sunnyguduru** [comment]
> If you are publishing multiple architectures, it should be as a multi-arch image. That way, they can simply pull ldcli and it will work on their platform.
> 
> This is kinda message in goreleaser, but here's how it's done in Relay: (docker manifest) https://github.com/launchdarkly/ld-relay/blob/v8/.goreleaser.yml#L122

I see. I was wondering what that was

File: .goreleaser.yaml [hunk 1]
```diff
@@ -25,7 +56,7 @@ brews:
       name: homebrew-tap
       token: "{{ .Env.GITHUB_TOKEN }}"
 
-    folder: Formula
+    directory: Formula
```
Comment (sunnyguduru) [inline .goreleaser.yaml:59]: `folder` was deprecated in favor of `directory`

**Review by cwaldren-ld — APPROVED** [review]
Looks good. I would recommend, for times like this, that you add a ["manual publish"](https://github.com/launchdarkly/ld-relay/blob/v8/.github/workflows/manual-publish.yml) github workflow.

Essentially, it would allow you to run the publish step independently of Release Please. That's super useful if an automated release fails to release to Brew/Docker/whatever for any reason. 

Then, you can simply run the manual workflow, point it to the release tag, and try again. 

In this scenario, it would be useful because you could point it at this branch and test it out before merging. (Although since it's docker, you'd need to temporarily modify all the image names to have a "-beta.1" suffix or something similar, since they aren't "official" yet.)

**Pull request merged**